src/test/kotlin/racingcar/domain/CarTest.kt

```diff
@@ -33,33 +15,32 @@ internal class CarTest {
 
     @Test
     fun `같은 거리인지 체크 다른 거리일 경우`() {
-        val car = Car(Name("test"))
-        val movedCar = car.move(Number(4))
+        val car = Car(Name("test"), Distance((1)))
 
         assertThat(car.isSameDistance(Distance(2))).isFalse()
     }
 
-    companion object {
-        @JvmStatic
-        fun overFour(): Stream<Number> {
-            return Stream.of(
-                Number(4),
-                Number(5),
-                Number(6),
-                Number(7),
-                Number(8),
-                Number(9)
-            )
+    @Test
+    fun `이동이 가능한 경우 거리가 바뀐다`() {
+        val car = Car(Name("test"), Distance((1)))
+        val moveStrategy = object : MoveStrategy {
+            override fun movable() = true
         }
 
-        @JvmStatic
-        fun underFour(): Stream<Number> {
-            return Stream.of(
-                Number(0),
-                Number(1),
-                Number(2),
-                Number(3)
-            )
+        val movedCar = car.move(moveStrategy)
+        assertThat(movedCar.distance).isNotEqualTo(car.distance)
+        assertThat(movedCar.distance).isGreaterThan(car.distance)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThat(movedCar.distance).isEqualTo(Distance(2))
```